plugin/storage/integration/cassandra_test.go

```diff
@@ -71,6 +71,10 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 	require.NoError(t, err)
 	s.SpanReader, err = f.CreateSpanReader()
 	require.NoError(t, err)
+	s.ArchiveSpanReader, err = f.CreateArchiveSpanReader()
```

Comment:
you need to pass flags to enable archive storage in L65

Reply:
Done, PTAL if the flags are passed correctly.